packages/preferences-persistence/src/create/index.js

```diff
@@ -83,26 +83,28 @@ export default function create( {
 			JSON.stringify( dataWithTimestamp )
 		);
 
-		// The user meta endpoint seems susceptible to errors when consecutive
-		// requests are made in quick succession. Ensure there's a gap between
-		// any consecutive requests.
-		//
-		// Catch and do nothing with errors from the REST API.
-		debouncedApiFetch( {
-			path: '/wp/v2/users/me',
-			method: 'PUT',
-			// `keepalive` will still send the request in the background,
-			// even when a browser unload event might interrupt it.
-			// This should hopefully make things more resilient.
-			// This does have a size limit of 64kb, but the data is usually
-			// much less.
-			keepalive: true,
-			data: {
-				meta: {
-					persisted_preferences: dataWithTimestamp,
-				},
-			},
-		} ).catch( () => {} );
+		debounce(
+			() =>
+				apiFetch( {
+					path: '/wp/v2/users/me',
+					method: 'PUT',
+					// `keepalive` will still send the request in the background,
+					// even when a browser unload event might interrupt it.
+					// This should hopefully make things more resilient.
+					// This does have a size limit of 64kb, but the data is usually
+					// much less.
+					keepalive: true,
+					data: {
+						meta: {
+							persisted_preferences: dataWithTimestamp,
+						},
+					},
+				} ).catch( () => {} ),
+			{
+				delayMS: isExpensive ? 60000 : requestDebounceMS,
```

Comment:
Should `60000` be configurable via a param passed to `create` same as `requestDebounceMS` is?